source/agora/crypto/ECC.d

```diff
@@ -168,7 +168,7 @@ public struct Scalar
 
     nothrow @nogc:
 
-    private this (typeof(this.data) data)
+    private this (typeof(this.data) data) pure inout
```

Comment:
Can I get any information(like web pages) about `inout` used as a function attribute? It's always confusing to me. :)

Reply:
This is quite a good description: https://forum.dlang.org/thread/d5ovc1$2rb$1@digitaldaemon.com

Reply:
Oh no, no, that's quite different :D

Reply:
Also http://www.prowiki.org/wiki4d/wiki.cgi?FunctionParameterAttributes

Reply:
As mentioned in Slack here is correct link:- https://dlang.org/spec/function.html#inout-functions